Classes/Controllers/PBGitHistoryController.m

```diff
@@ -401,6 +401,21 @@ - (IBAction)selectPrevious:(id)sender
 	[searchController selectPreviousResult];
 }
 
+- (IBAction) selectParentCommit:(id)sender
+{
+	NSArray *selectedObjects = commitController.selectedObjects;
+	if (selectedObjects.count != 1) return;
+
+	PBGitCommit *selectedCommit = selectedObjects[0];
+
+	NSArray <GTOID *> *parents = selectedCommit.parents;
+	/* TODO: This is a merge commit. It would be nice to choose the parent with
+	 * the most commits, but for now we will use whatever commit is our first parent.
+	 */
+
```

Comment:
Just wondering whether that heuristic may be unfortunate as it can be hard to tell which branch has the most commits for the user – especially if you merge a branch that was forked earlier on.

As far as I can tell, when merging a forked branch back into your main branch you’ll get a commit where the first parent is the main branch, meaning that the heuristic you chose should give the result I’d consider as the most intuitive in that standard case.